test/controllers/returns/ImportedPlasticPackagingControllerSpec.scala

```diff
@@ -102,13 +112,11 @@ class ImportedPlasticPackagingControllerSpec extends SpecBase with MockitoSugar
 
     "must redirect to the next page when valid data is submitted" in {
 
-      val mockCacheConnector = mock[CacheConnector]
-      when(mockCacheConnector.set(any(), any())(any())) thenReturn Future.successful(mockResponse)
-      val returnsJourneyNavigator = mock[ReturnsJourneyNavigator]
-      when(returnsJourneyNavigator.importedPlasticPackagingRoute(any(), any(), any())) thenReturn onwardRoute
+      // user answers responds with "answer not changed"
```

Comment:
Should this comment form part of the test name rather than being a comment?

Reply:
:eyes: